phys2bids/phys2bids.py

```diff
@@ -176,7 +176,7 @@ def use_heuristic(heur_file, sub, ses, filename, outdir, record_label=''):
 
 def phys2bids(filename, info=False, indir='.', outdir='.', heur_file=None,
               sub=None, ses=None, chtrig=0, chsel=None, num_timepoints_expected=0,
-              tr=1, thr=2.5, ch_name=[], chplot='', debug=False, quiet=False):
+              tr=1, thr=2.5, ch_name=[], chplot='', debug=False, quiet=False, std_option=0):
```

Comment:
```suggestion
              tr=1, thr=2.5, ch_name=[], chplot='', debug=False, quiet=False, std_option=False):
```

I know that `0` and `1` are identical to `False` and `True` for all intents and purposes in Python, but since you currently have this set up as a binary option I think it would be better to use `std_option=False`.